integration-testing/package.json

```diff
@@ -3,20 +3,19 @@
     "version": "1.0.0",
     "description": "Integration tests for the bods-integrated-data platform",
     "scripts": {
-        "test": "playwright test",
-        "test:dev": "STAGE=dev playwright test",
-        "test:test": "STAGE=test playwright test"
+        "test": "playwright test"
```

Comment:
why were the test:dev and test:test scripts removed? I think we might need to keep them or update the make command `run-integration-tests-%` to set STAGE as an env var.

Reply:
ahh that was from testing changes